cmd/server/main.go

```
@@ -0,0 +1,66 @@
+	err := godotenv.Load()
+	if err != nil {
+		panic("Error loading .env file")
+	}
+
+	// DynamoDBの設定
+	endpoint := os.Getenv("DYNAMO_ENDPOINT")
+	region := os.Getenv("AWS_REGION")
+	if region == "" {
+		region = "ap-northeast-1"
+	}
+	cfg, err := config.LoadDefaultConfig(context.Background(),
```

golangci-lint (1.64.8)

ineffectual assignment to err

(ineffassign)